### PR TITLE
Update Plexus.csv

### PR DESCRIPTION
**Pull request by GidCat**

### CODE REVIEW

File: src/data/Plexus.csv [hunk 1]
```diff
@@ -1,4 +1,7 @@
 Application,Month,Year,Rating (1-4),Notes,MicroG Rating (1-4),MicroG Notes
+Abrecombie & Fitch,July,2021,X,X,4,No reported issues
+adidas,July,2021,X,X,4,No reported issues
+American Eagle AE + Aerie,July,2021,X,X,4,No reported issues
```
Comment (tomkonidas) [inline src/data/Plexus.csv:4]: Can we order these alphabetically in the list with the other A's

**Review by tomkonidas — CHANGES_REQUESTED** [review]

File: src/data/Plexus.csv (hunk 1 above)
Comment (GidCat) [reply src/data/Plexus.csv]: All done.

**Review by GidCat — COMMENTED** [review]
Orderd Abrecombie, adidas, and American Eagle AE + Aerie alphabetically.

File: src/data/Plexus.csv [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Application,Month,Year,Rating (1-4),Notes,MicroG Rating (1-4),MicroG Notes
+
```
Comment (tomkonidas) [inline src/data/Plexus.csv:2]:
```suggestion
```

**Review by tomkonidas — APPROVED** [review]

**Pull request merged**